### DISCUSSION TITLE
Mehrere homematicip_local-Konfigurationseinträge mit derselben eindeutigen ID seit Core-Update auf 2024.11.0

### DISCUSSION THREAD

**Discussion by anderl1969** [SukramJ/aiohomematic#1823] [Q&A] [answered]
Hallo zusammen,

seit dem Update von Home-Assistant Core auf Version 2024.11.0 bekomme ich eine Fehlermeldung bzgl. Homematic IP:

```
Mehrere homematicip_local-Konfigurationseinträge mit derselben eindeutigen ID
Fehler ⸱ Von Home Assistant Core Integration gemeldet

Dies funktioniert nicht mehr in Version 2025.11.0. Bitte behebe dies vor dem Upgrade.

Es gibt mehrere homematicip_local-Konfigurationseinträge mit derselben eindeutigen ID.
Die Konfigurationseinträge heißen 'CCU3', 'raspmatic/192.168.1.10'

Um diesen Fehler zu beheben, konfiguriere die Integration und entferne alle bis auf eines der Duplikate.

Hinweis: Nach dem Entfernen dieser Duplikate kann eine weitere Gruppe von Duplikaten sichtbar werden.
```

Um überhaupt mal eine Idee zu bekommen, wo ich suchen soll, musste ich die Sprache auf Englisch umstellen:
```
Multiple homematicip_local config entries with same unique ID
Error ⸱ Reported by Home Assistant Core Integration

There are multiple homematicip_local config entries with the same unique ID.
The config entries are named 'CCU3', 'raspmatic/192.168.1.10'.

To fix this error, configure the integration and remove all except one of the duplicates.

Note: Another group of duplicates may be revealed after removing these duplicates.
```

Bei meiner weiteren Recherche bin ich dann auf die Datei `/homeassistant/.storage/core.config_entries` gestoßen:

```json
{
  "version": 1,
  "minor_version": 4,
  "key": "core.config_entries",
  "data": {
    "entries": [
      ...
      {"created_at":"1970-01-01T00:00:00+00:00","data":{"advanced_config":{},"host":"raspmatic.mydomain.local","instance_name":"CCU3","interface":{"BidCos-RF":{"port":42001},"HmIP-RF":{"port":42010},"VirtualDevices":{"path":"/groups","port":49292}},"password":"xxxxxxxxxxxxxxxxxxx","tls":true,"username":"myusername","verify_tls":false},"disabled_by":null,"discovery_keys":{"ssdp":[{"domain":"ssdp","key":"uuid:upnp-BasicDevice-1_0-3333FFFFFFFFFF1234567890","version":1},{"domain":"ssdp","key":"uuid:upnp-BasicDevice-1_0-","version":1}]},"domain":"homematicip_local","entry_id":"222bbeee991dd33ccccc7cc66666bb11","minor_version":1,"modified_at":"2024-11-06T21:19:48.992516+00:00","options":{},"pref_disable_new_entities":false,"pref_disable_polling":false,"source":"ssdp","title":"CCU3","unique_id":"1234567890","version":7},
      {"created_at":"1970-01-01T00:00:00+00:00","data":{},"disabled_by":null,"discovery_keys":{},"domain":"homematicip_local","entry_id":"3bbb4444e3333e0c446ee11ee999999c","minor_version":1,"modified_at":"1970-01-01T00:00:00+00:00","options":{},"pref_disable_new_entities":false,"pref_disable_polling":false,"source":"ignore","title":"raspmatic/192.168.1.10","unique_id":"1234567890","version":2},
      ...
    ]
  }
```

Und da stehen die beiden Einträge drin mit der gleichen unique_id!

Aber darf ich da einfach einen löschen? Wenn ja, welchen? Irgendwo hatte ich gelesen, dass man in den Dateiein unter `.storage/` nicht rumpfuschen soll :(

**Comment by SukramJ** [comment]
Keine Ahnung. Mach ein Backup und probiere es aus.

**Reply by SukramJ** [reply, marked as answer]
Der Untere Eintrag ist version 2, und somit viel älter. Ich würde den löschen.

**Reply by anderl1969** [reply]
Woran erkennst Du das, dass der untere Eintrag der ältere ist? Der Timestamp ist ja bei beiden nicht so aussagekräftig. An der version? 

Nachdem ich mit `ha core stop` Home-Assistant gestoppt habe, habe ich mich getraut, und den unteren Eintrag gelöscht.
Nachdem ich HA wieder gestartet habe, war die Fehlermeldung weg. Die Homematic Integration scheint auch fehlerfrei zu arbeiten. Mal noch ein paar Tage beobachten, aber ich bin zuversichtlich.

Da bei mir sowohl HA als Homematic in Proxmox VMs laufen, bin ich mit Backups bestens versorgt. :)

**Reply by SukramJ** [reply]
Version 2 ist alter als Version 7

**Comment by centertivevier** [comment]
Hi,
habe dasselbe Problem. Bei mir stand ein Eintrag mit Version 1. Habe es gelöscht. Komischerweise hab ich den Eintrag hier nicht gefunden über die Suche. Wer weiß. Trotzdem Danke für die schnelle Hilfe.

**Reply by anderl1969** [reply]
Hmm, hatte extra die deutsche und englische Fehlermeldung reinkopiert, damit es leichter zu finden ist...
